tv_apps/smart-system/js/app_install_manager.js

```diff
@@ -408,11 +427,12 @@ var AppInstallManager = {
     var manifest = app.manifest || app.updateManifest;
     var appManifest = new ManifestHelper(manifest);
     var name = appManifest.name;
-    var msg = {
-      id: 'app-install-success',
-      args: { appName: name }
-    };
-    this.systemBanner.show(msg);
+    this.systemBanner.show({
+      id: 'app-add-success',
```

Comment:
I think you shouldn't amend this message directly since other sites may trigger "real" install request as well.